codelabs/Utopia Wholesale Codelab/Holder/0-Get Credential.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Get Credential
+sidebar_position: 0
+---
```

Comment:
why is this empty?

Reply:
Hi @hanluOMH, based on our last conversation, it looks like the provisioning steps are included in the [[Provisioning Section](https://docs.google.com/document/d/1PUyO0MW0R0nO2rKufa-t7XYda45QjObj07Z1XlwR-Ak/edit?tab=t.ya18sx84954j#heading=h.az40ixii23ys)](https://docs.google.com/document/d/1PUyO0MW0R0nO2rKufa-t7XYda45QjObj07Z1XlwR-Ak/edit?tab=t.ya18sx84954j#heading=h.az40ixii23ys). If that’s the case, could you please add a comment noting that these steps are WIP and coming soon?

Also, would it make sense to rename `Get Credential` to `Obtaining a credential`, which in most cases could be provisioned by the server/backend app? Thanks!